src/ocamlorg_frontend/css/partials/cards.css

```
@@ -7,5 +7,5 @@
 }
 
 .card {
-  @apply bg-white drop-shadow hover:shadow-[0px_0px_12px_0px_#d5d5d5] rounded-2xl border;
+  @apply bg-white drop-shadow hover:shadow-[0px_0px_12px_0px_#00000080] rounded-2xl border;
```

```suggestion
  @apply bg-white drop-shadow hover:shadow-[0px_0px_12px_0px_#00000080] rounded-2xl border;
}

.dark-card {
  @apply bg-dark-card drop-shadow hover:shadow-none rounded-2xl border;
```

We can use `card dark:dark-card` classes when we wrap these utilities with `@layer utilities` like described here: https://stackoverflow.com/questions/75889347/how-can-i-use-a-custom-class-with-tailwind-css-for-dark-mode